Makefile

```diff
@@ -12,17 +12,29 @@ TESTFLAGS :=
 # Required for globs to work correctly
 SHELL=/usr/bin/env bash
 
+GIT_COMMIT  ?= $(shell git rev-parse --short HEAD)
+
+HAS_GLIDE := $(shell command -v glide;)
+HAS_DEP := $(shell command -v dep;)
+HAS_OPERATOR_SDK := $(shell command -v operator-sdk)
+
 .PHONY: build check-env
 build: check-env
-	operator-sdk build $(OPERATOR_IMAGE)
+	operator-sdk build $(OPERATOR_IMAGE):$(OPERATOR_IMAGE_TAG)
+	operator-sdk build $(OPERATOR_IMAGE):$(GIT_COMMIT)
 
 push:
-	docker push $(OPERATOR_IMAGE)
+	docker push $(OPERATOR_IMAGE):$(OPERATOR_IMAGE_TAG)
```

Comment:
You *could* do a `for tag in $(OPERATOR_IMAGE_TAG) $(GIT_COMMIT)` here, but I'm not sure it is a real improvement and mostly because I tend to over-DRY things myself.